docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3'
+
+services:
+  db:
+    image: postgres:10-alpine
+    ports:
+      - "5432:5432"
+    volumes:
+      - ./var/data/pgsql/:/var/lib/postgresql
+    env_file: '.docker/paste/postgres_env'
```

Comment:
that's going to create a single user, you should create a migration user (the superadmin) and an application user (without create/drop/alter) to mitigate sql injections.

Reply:
that's not a concern for the initial implementation